CONTRIBUTING.md

```diff
@@ -32,6 +32,18 @@ submitting a change-list (CL).
 In general, we're happier reviewing many small PRs over fewer, larger changes. Please try to
 submit [small CLs](https://google.github.io/eng-practices/review/developer/small-cls.html).
 
+## Developer Meetings
+
+To facilitate internal and community support of weather tools, we invite you to join our regular developer meetings.
+Since our contributors are in a wide range of time zones, we offer two fortnightly meetings: Wednesdays at 8:30 AM PST
+and Fridays at 3:00 PM PST. Feel free to attend to only one meeting every two weeks, whichever best works with your
+schedule. Our first open meeting will occur on Friday, January 28th, 2022 (the first morning meeting will be held on
+Wednesday, February 2nd, 2022).
+
+[Here is a calendar](https://calendar.google.com/calendar/u/0?cid=Y19uYzJhNXVxMzE1ZnAzb2U1bWUwMGw2NTBxOEBncm91cC5jYWxlbmRhci5nb29nbGUuY29t)
+of our meeting schedule,
+and [here are our meeting notes](https://docs.google.com/document/d/1V2iISi-ZZNmrcImrPV7UYQGFNAe9XQ_-TB8iPggfmbo/edit).
```

Comment:
This is public and editable by anyone. I'm not sure that's a good idea tbh. Maybe make it publicly viewable but only a limited set of editors and maybe some commenters?

Reply:
I think it's ok to keep these totally public. I'd like to encourage open collaboration for this project, and want to have as little friction as possible for others to see what we have planned. 

This is more or less following the pangeo model: https://github.com/pangeo-forge/pangeo-forge-recipes#get-in-touch